### PR TITLE
chore(ci): test on PHP 8.2

### PR DESCRIPTION
**Pull request by kbond**
Blocked by:
- [x] https://github.com/laminas/laminas-code/issues/142

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0, 8.1]
+        php: [8.0, 8.1, 8.2]
```
Comment (kbond) [inline .github/workflows/ci.yml:16]: @nikophil, did I miss a set of include permutations below in #359? Or, how should we handle so we don't explode the matrix?

Comment (kbond) [reply .github/workflows/ci.yml:16]: Maybe just?
```diff
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 0, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 0, use-dama: 0, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 1, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 1, use-dama: 0, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 0, use-dama: 1, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 0, use-dama: 1, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 0, use-odm: 1, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 0, use-odm: 1, use-dama: 0, deps: highest}
```

Comment (nikophil) [reply .github/workflows/ci.yml:16]: yes, this way the highest and the lowest [php, sf, deps] are tested with all permutations
while all others are just tested with dama:1 orm:1 odm:1

**Comment by Ocramius** [comment]
`laminas/laminas-code:4.8.0` supports PHP 8.2 now.

**Comment by kbond** [comment]
Thanks @Ocramius!

**Pull request merged**